packages/runtime/src/constants.ts

```diff
@@ -7,18 +9,29 @@ export const HANDLER_FUNCTION_TITLE = 'Next.js SSR handler'
 export const ODB_FUNCTION_TITLE = 'Next.js ISR handler'
 export const IMAGE_FUNCTION_TITLE = 'next/image handler'
 // These are paths in .next that shouldn't be publicly accessible
-export const HIDDEN_PATHS = [
-  '/cache/*',
-  '/server/*',
-  '/serverless/*',
-  '/trace',
-  '/traces',
-  '/routes-manifest.json',
-  '/build-manifest.json',
-  '/prerender-manifest.json',
-  '/react-loadable-manifest.json',
-  '/BUILD_ID',
-]
+export const HIDDEN_PATHS = destr(process.env.NEXT_KEEP_METADATA_FILES)
```

Comment:
Since it's for E2E/CI only (primarily), we already have a `CI` environment variable. See a usage of it here, https://github.com/netlify/next-runtime/blob/82c1ea371bbe2194058264d7cfad912343334809/package.json#L19.

```suggestion
export const HIDDEN_PATHS = destr(process.env.CI)
```

Reply:
`CI` env var is `true` on Netlify platform, so we can't rely on that, otherwise we won't actually delete those meta files :(